helios/pipeViewer/transactionsearch/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+project(TransactionSearch)
+
+add_executable(transactionsearch src/transaction_search.cpp)
+add_dependencies(transactionsearch sparta simdb)
+
+set(Boost_USE_STATIC_LIBS OFF)
+set(BUILD_SHARED_LIBS ON)
+
+find_package(Boost 1.49.0 REQUIRED COMPONENTS regex)
```

Comment:
Ubuntu18.04 has boost1.65, which is pretty common, is 1.49.0 needed specifically?

Reply:
Ok, I can move the base.  Forgot it was there. 

Fixed.